crates/dojo-core/src/interfaces.cairo

```diff
@@ -18,6 +18,7 @@ trait IWorld {
     );
     fn entities(component: ShortString, partition: u250) -> Array::<u250>;
     fn set_executor(contract_address: starknet::ContractAddress);
+    fn delete_entity(component: ShortString, query: Query);
```

Comment:
Looks like we need to import `Query` here too

Reply:
done